resources/graphql/authorization.graphql

```diff
@@ -12,7 +12,7 @@ type User {
 }
 
 type UserInfo {
-  userID: String # MongoDB native ID or UUID
+  _id: String # MongoDB native ID or UUID
```

Comment:
Verify that this change is ok with Ming and the FE team. I am fine with making this change as long as they are as well